examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+            "Successfully installed frozendict-1.2 kaldi-io-0.9.4 marshmallow-3.6.0 nemo-toolkit-0.10.0b10 num2words-0.5.10 onnx-1.7.0 onnxruntime-1.2.0 ruamel.yaml-0.16.10 ruamel.yaml.clib-0.2.0 soundfile-0.10.3.post1 sox-1.3.7 tensorboardX-2.0 torch-stft-0.1.4 unidecode-1.1.1\n",
+            "Requirement already satisfied: unidecode in /usr/local/lib/python3.6/dist-packages (1.1.1)\n",
+            "--2020-05-11 02:14:48--  https://raw.githubusercontent.com/NVIDIA/NeMo/master/examples/speaker_recognition/configs/quartznet_spkr_3x2x512_xvector.yaml\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 1789 (1.7K) [text/plain]\n",
+            "Saving to: ‘configs/quartznet_spkr_3x2x512_xvector.yaml’\n",
+            "\n",
+            "quartznet_spkr_3x2x 100%[===================>]   1.75K  --.-KB/s    in 0s      \n",
+            "\n",
+            "2020-05-11 02:14:48 (28.4 MB/s) - ‘configs/quartznet_spkr_3x2x512_xvector.yaml’ saved [1789/1789]\n",
+            "\n",
+            "--2020-05-11 02:14:48--  https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/scp_to_manifest.py\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 1742 (1.7K) [text/plain]\n",
+            "Saving to: ‘scp_to_manifest.py’\n",
+            "\n",
+            "scp_to_manifest.py  100%[===================>]   1.70K  --.-KB/s    in 0s      \n",
+            "\n",
+            "2020-05-11 02:14:48 (42.4 MB/s) - ‘scp_to_manifest.py’ saved [1742/1742]\n",
+            "\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VgTR8CMlxu3p",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **SPEAKER RECOGNITION** \n",
+        "\n",
+        "Speaker Recognition (SR) is an broad research area which solves two major tasks: speaker identification (who is speaking?) and speaker verification (is the speaker who she claims to be?). In this work, we focmus on the far-field, text-independent speaker recognition when the identity of the speaker is based on how speech is spoken, not necessarily in what is being said. Typically such SR systems operate on unconstrained speech utterances, which are converted into vector of fixed length, called speaker embedding. Speaker embedding is also used in automatic speech recognition (ASR) and speech synthesis.\n",
```

Comment:
which are converted into vector of fixed length, called speaker embedding. Speaker embedding is also used in automatic speech recognition (ASR) and speech synthesis.
->
which are converted into vectors of fixed length, called speaker embeddings. Speaker embeddings are also used in automatic speech recognition (ASR) and speech synthesis.

Reply:
Done